docs/upgrade.html

```diff
@@ -63,59 +63,6 @@ <h4><a id="upgrade_2_0_0" href="#upgrade_2_0_0">Upgrading from 0.8.x, 0.9.x, 0.1
         Hot-swaping the jar-file only might not work.</li>
 </ol>
 
-<h5><a id="upgrade_200_notable" href="#upgrade_200_notable">Notable changes in 2.0.0</a></h5>
```

Comment:
This section was added by mistake because 1.2.0 is now 2.0.0; piggy back the removal of it here.